package.json

```diff
@@ -18,22 +18,7 @@
   "keywords": [],
   "dependencies": {},
   "devDependencies": {
-    "coveralls": "^2.11.1",
-    "gulp": "^3.8.8",
-    "gulp-bump": "^0.1.11",
-    "gulp-istanbul": "^0.3.0",
-    "gulp-jscs": "^1.1.2",
-    "gulp-jshint": "^1.8.4",
-    "gulp-load-plugins": "^0.6.0",
-    "gulp-mocha": "^1.1.0",
-    "gulp-plumber": "^0.6.5",
-    "gulp-util": "^3.0.1",
-    "jsdom": "^6.3.0",
-    "jshint-stylish": "^0.4.0"
```

Comment:
😊 you're right, linting doesn't harm anybody. Added see #4970348 Thanks for raising your concern